sale_order_partner_no_autofollow/models/res_config_settings.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    sale_partner_no_autofollow = fields.Boolean(
+        config_parameter="sale_order.customer_no_autofollow",
```

Comment:
config_parameter must be contained "<module_name>.<config_field>"

Reply:
Ok, I'll fix it)